docs/install/install.md

```diff
@@ -183,7 +185,7 @@ Configuration can be done by both environment variable(conf/zeppelin-env.sh) and
     <td>org.apache.zeppelin.spark.SparkInterpreter,<br />org.apache.zeppelin.spark.PySparkInterpreter,<br />org.apache.zeppelin.spark.SparkSqlInterpreter,<br />org.apache.zeppelin.spark.DepInterpreter,<br />org.apache.zeppelin.markdown.Markdown,<br />org.apache.zeppelin.shell.ShellInterpreter,<br />org.apache.zeppelin.hive.HiveInterpreter<br />
     ...
     </td>
-    <td>Comma separated interpreter configurations [Class]. First interpreter become a default</td>
+    <td>Comma separated interpreter configurations [Class] <br /> The first interpreter will be a default value.</td>
```

Comment:
Actually, this really means the first interpreter on this list is the default interpreter (which means it's the one used if %name is not entered in the notebook paragraph)
I do think maybe we should explain what the default interpreter is...